common/lib/xmodule/xmodule/js/spec/video/html5_video_spec.js

```diff
@@ -322,6 +323,22 @@
             });
         });
 
+        describe('non-hls encoding', function() {
+            beforeEach(function(done) {
+                state = jasmine.initializePlayer('video_html5.html');
+                done();
```

Comment:
specs will not start until this done callback is called, just to ensure `beforeEach` is completed execution.